firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -394,7 +394,7 @@ public TokenResult generateAuthToken(
         logBadConfigError();
 
         return TokenResult.builder().setResponseCode(TokenResult.ResponseCode.BAD_CONFIG).build();
-      } catch (IOException ignored) {
+      } catch (AssertionError | IOException ignored) {
```

Comment:
Can we add a TODO to remove this code duplication (from above)?

Reply:
Added a TODO and created a tracker https://b.corp.google.com/issues/166168291 for removing code duplication. Thanks.